sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Employee.java

```diff
@@ -0,0 +1,41 @@
+package com.exadel.sandbox.team5.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+@Entity
+@Table(name = "employee")
```

Comment:
I think that it will be better to add additional constraint for login field at least:
```
@Table(uniqueConstraints = @UniqueConstraint(columnNames = {"login"}))
```